packages/@okta/vuepress-site/docs/guides/third-party-risk-integration/overview/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Third-Party Risk Provider Integration Overview
+---
+
+<ApiLifecycle access="beta" /> This is a beta feature guide.
+
+The Okta Risk Engine evaluates authentication attempts by reviewing the risk score of the sign-in based on context and historical data. Using Okta Risk APIs, third-party Risk Providers can integrate with the Okta Risk Engine using a standard Okta service application. The third-party Risk Provider can send risk signals, or events, which can be used when calculating the authentication risk based on the risk policy configured in the Okta org. The risk signals are additionally logged as part of the System Log.
```

Comment:
In "risk signals, or events" do you mean System Log events? Because if so you should be explicit.

Reply:
Good point. I meant to get this clarified. The third-party provider sends "signals" or "events", and are used interchangeably in the specs. I like "signals" to avoid confusion with Okta System Log events; however, our API is called Risk Events, which is used to send said "signals" or "events". I'll open up the discussion on Slack.

Reply:
As per Slack convo, standardizing on "risk events"